bike_rent/models/sale.py

```diff
@@ -11,17 +11,23 @@ def action_confirm(self):
         # <AndriusD code
         def filter_rests(rec):
             product = rec.product_id
-            return product.is_bike and product.type == 'service'
-
-        for record in self.order_line.filtered(filter_rests):  # /AndriusD code>
-            rent_rec = self.env['bike.rent'].create({
-                'notes': record.name,
-                'price': record.price_total,
-                'partner_id': record.order_partner_id.id,
-                'bike_id': record.product_id.id,
-                'rent_time': record.product_id.rent_duration,
-            })
-            record.bike_rent_id = rent_rec.id
+            return product.is_bike and product.type == 'service'  # /AndriusD code>
+
+        def create_bike_rent_objects(so_record):
+            for record_line in so_record.product_id.rented_bike_ids:
```

Comment:
Whoa, why are you creating `bike.rent` for every rent related bike? Which task asked for this functionality?